content/home-2019/card_2/en-US.md

```diff
@@ -1,11 +1,11 @@
 ---
 # card 2
-title: "Hacking the STEM syllabus"
-image: "stem.png"
-link_url: "https://www.weforum.org/agenda/2018/12/hacking-the-stem-syllabus/"
-tag_label: "Op-Ed: Mitchell Baker"
+title: "Celebrating women and the web"
+image: "women-web.png"
+link_url: "https://blog.mozilla.org/internetcitizen/2019/03/08/international-womens-day-2019/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card"
```

Comment:
Note: this blog post will be published on March 8; we'll need to make sure it's live before we push the card updates to production.